netlify.toml

```diff
@@ -5,21 +5,9 @@
   for = "/*"
   [headers.values]
     Content-Security-Policy = '''
-      default-src 'self' https://*.netlify.app;
-      connect-src 'self' data: https://keybase.io https://lcd.nylira.net;
+      default-src 'self' 'unsafe-inline' https://*.netlify.app;
```

Comment:
this doesn't look right, why make this a default?

Reply:
It has to do with Jordan's proposed simplification: https://github.com/luniehq/lunie-light/pull/292#issuecomment-734872156

Reply:
I think you misinterpreted him. He meant "default-src 'self'" should be enough. But it isn't. This will allow anything added unsafe-inline. Please remove and restore the previous fine-grained controls.